build_targets: Add zippo

chematelegram opened the pull request.

chematelegram committed:
build_targets: Add zippo

gwolf2u reviewed (changes requested):
Please fork to crDroid Gitlab or Github
https://github.com/crdroidandroid/android_device_lenovo_zippo/blob/13.0/crdroid.dependencies#L7#L10
Same here
https://github.com/crdroidandroid/android_device_lenovo_zippo/blob/13.0/crdroid.dependencies#L21#L24

You may also remove lineage dependencies
https://github.com/crdroidandroid/android_device_lenovo_sm8150-common/blob/13.0/lineage.dependencies

chematelegram commented:
Hey G., first thanks for the attention and the detailed instructions.

Following your indications I've pushed my vendor to crdroidandroid and changed it in dependencies.

Also I've removed lineage.dependencies from device_lenovo_sm8150-common .

Regarding the custom clang, I have removed it from dependencies since I've switched to clang-latest as you can see in the commit history for device_lenovo_zippo .
